swap/swap_test.go

```diff
@@ -136,6 +136,124 @@ func testBalances(t *testing.T, swap *Swap, expectedBalances map[enode.ID]int64)
 	}
 }
 
+type storeKeysTestCases struct {
+	nodeID                    enode.ID
+	expectedBalanceKey        string
+	expectedSentChequeKey     string
+	expectedReceivedChequeKey string
+}
+
+func TestStoreKeys(t *testing.T) {
```

Comment:
Also, this is actually a unit test for the `StateStore` keys function.
I wonder if it should not be moved over there

Reply:
The tests are not completely agnostic to business logic, since they use the balance and cheque prefixes constants.

Do you still think they should be moved to `dbstore_test.go` despite this?

Reply:
As per https://github.com/ethersphere/swarm/pull/1610#discussion_r308292596, this design may change, so we will address this in https://github.com/ethersphere/swarm/pull/1554